aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -357,6 +380,18 @@ public class AwsProperties implements Serializable {
    */
   public static final String LAKE_FORMATION_DB_NAME = "lakeformation.db-name";
 
+  /**
+   * Used by {@link GlueCatalog}.
+   * The table name is used as a part of table specific catalog properties.
+   */
+  public static final String S3_TABLE_SPECIFIC_WRITE_TAGS_TABLE_NAME = "iceberg.table-name";
```

Comment:
looks like this should be moved to under `S3_WRITE_TAGS_PREFIX`? And we can have a simpler name like `S3_TAG_TABLE_NAME`

Reply:
Thanks, I have made the change.